public/docs/ts/latest/guide/production-deploy.jade

```diff
@@ -0,0 +1,480 @@
+    * [Routed apps must fallback to `index.html`](#fallback)  
+    * [CORS: requesting services from a different server](#cors)
+
+a#overview
+.l-main-section
+:marked
+  ## Overview
+
+  This guide describes techniques for preparing and deploying an Angular application to a server running remotely.
+  The techniques progress from _easy but suboptimal_ to _more optimal and more involved_.
+
+  * The [simple way](#dev-deploy "Simplest deployment possible") is to copy the development environment to the server.
+
+  * [_Ahead of Time_ compilation (AOT)](#aot "AOT Compilation") is the first of 
+  [several optimization strategies](#optimize). 
+  You'll also want to read the [detailed instructions in the AOT Cookbook](aot-compiler.html "AOT Cookbook").
+  
+  * [Webpack](#webpack "Webpack Optimization") is a popular general purpose packaging tool with a rich ecosystem, including plugins for AOT.
+  The Angular [webpack guide](webpack.html "Webpack: an introduction") can get you started and
+  _this_ page provides additional optimization advice, but you'll probably have to learn more about webpack on your own.
+
+  * The [Angular configuration](#angular-configuration "Angular configuration") section calls attention to
+  specific client application changes that could improve performance.
+
+  * The [Server configuration](#server-configuration "Server configuration") section describes
+  server-side changes that may be necessary, _no matter how you deploy the application_.
+
+.l-main-section
+a#dev-deploy
+:marked
+  ## Simplest deployment possible 
+
+  The simplest way to deploy the app is to publish it to a web server
+  directly out of the development environment.
+  It's already running locally. You're just copying it, almost _as is_, 
+  to a non-local server that others can reach.
+
+  1. Copy _everything_ (or [_almost_ everything](#node-modules "Loading npm packages from the web")) 
+  from the local project folder to a folder on the server.
+
+  1. If you're serving the app out of a subfolder,
+  edit a version of `index.html` to set the `<base href>` appropriately. 
+  For example, if the URL to `index.html` is `www.mysite.com/myapp/`, set the _base href_  to 
+  `<base href="/myapp/">`.
+  Otherwise, leave it alone.
+  [More on this below](#base-tag).
+
+  1. Configure the server to redirect requests for missing files to `index.html`.
+  [More on this below](#fallback).
+
+  1. Enable production mode as [described below](#enableprodmode) (optional).
+
+  That's all it takes to publish your app!
+
+a#node-modules
+:marked
+  ### Load npm packages from the web (SystemJS)
+
+  The `node_modules` folder of _npm packages_ is huge and slow to upload to the server.
+  It's typically 20,500+ files and 180+ MB.
+  The application itself requires a tiny fraction of that to run.
+
+  Load the _npm packages_ from the web instead.
+
+  (1) Make a copy of `index.html` for deployment and replace all `node_module` scripts
```

Comment:
I think here, we want to use SystemJS builder instead. UMD bundles are a less optimal option and straight out won't work with AoT